composer.json

```diff
@@ -17,10 +17,13 @@
 		"humanmade/gaussholder": "1.1.3",
 		"humanmade/aws-rekognition": "~0.1.7",
 		"humanmade/rename-images-command": "^0.1.0",
-		"humanmade/asset-manager-framework": "^0.11.0",
+		"humanmade/asset-manager-framework": "^0.12.0",
 		"humanmade/amf-wordpress": "^0.2.0"
 	},
 	"autoload": {
+		"classmap": [
+			"inc/"
```

Comment:
The `class-tachyon-provider.php` doesn't seem to get called for me on my local and throws a fatal undefined class error for `Tachyon_Provider`. Wondering if this isn't recursing?

Reply:
Think it’ll be your autoloader not generating properly. Try composer dump-autoload or composer update -W. If that doesn’t do it get rid of your local checkout of AMF and remove the symlink in the vendor directory then run the update again

Reply:
Okay, then this is tied to the fact that I can't actually `composer update -W` when I have this branch checked out because I get a composer error with the version conflict between `amf-wordpress` requiring `^0.11.0` and `altis/media` requiring `^0.12.0` of AMF.

Reply:
Ah right, I def need to fix that then.